src/v/crypto/tests/BUILD

```diff
@@ -168,7 +168,7 @@ redpanda_cc_gtest(
     cpu = 2,
     data = common_data_deps,
     defines = ["FIPS_MODULE_REQUIRED"],
-    env = common_env,
+    env = common_env | {"RP_FIXTURE_ENV": ""},
```

Comment:
just discussion: this is a bit awkward, right? like there is some aspect of fixture tests (whatever is testing `in_rp_fixture_test()`) that should drive the name of this environment variable, rather than it being a "fixture test"?

Reply:
Yeah, personally I am not a fan of this method to toggle behavior deep in our crypto libraries. In terms of naming yeah this was just blanket added to every fixture test in CMake, and the crypto library does different stuff based on the variable

Reply:
Yes it becomes a misnomer this way. It'll probably be easier to rename it when cmake/ctest goes.